docs/rules/no-useless-constructor.md

```diff
@@ -0,0 +1,56 @@
+# Disallow unnecessary constructor (no-useless-constructor)
+
+ES2015 provides a default constructor if one is not specified. As such, it is unnecessary to provide an empty constructor or one that simply delegates into its parent class.
```

Comment:
I added a couple of examples, but I don't know what else to add beyond what's already there.